sql/expression_resolver.go

```diff
@@ -368,16 +374,25 @@ func resolveExpression(e interface{}) (interface{}, error) {
 }
 
 func resolveColumnSpec(el *exprlist, isSparse bool) (*columnSpec, error) {
```

Comment:
I am using `expression_resolver` in `codegen.go` too, and seems `ColumnSpec` is only used for `DENSE/SPARSE` columns in ALPS, it'll be good to separate `ColumnSpec` and `FeatureColumn` structs.